### ISSUE TITLE
MnKAlphaDistribution should never return negative energy

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#112]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/112) by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

```
import mass
import numpy as np
distrib = mass.calibration.MnKAlphaDistribution()
np.random.seed(134)
np.amin(distrib.rvs(size=1000000))
```

On my computer that returns `-1557147.2023549364`. It really should never be negative, and I'm surprised it is more than say 100 eV from the line center.

That should never return a negative number, but it does.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Whoa! Me, too.

Of course, this comes about because the Lorentzian distribution has infinite extent. That may be unphysical, but I'm not sure what the right solution is. I guess you could do a while loop to re-generate all values that are negative.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fluorescence line distributions no longer produce negative energy.

Fixes #112 and tests for the fix.
